pages/common/source.md

```diff
@@ -0,0 +1,11 @@
+# Source
+
+> The source command can be used to load any shell file into the current shell script or a command prompt.
```

Comment:
"load any shell file into the current shell script" ? Not sure if that makes sense to me.